src/rp/util/logic.clj

```diff
@@ -0,0 +1,7 @@
+(ns rp.util.logic)
+
+(defn xor
+  [p q]
```

Comment:
It goes on to say "A chain of XORs—a XOR b XOR c XOR d (and so on)—is true whenever an odd number of the inputs are true and is false whenever an even number of inputs are true." I wonder if there's a certain bias there due to thinking of it as an operator (between two operands) instead of as a function of multiple operands.

If they are chained together in gate fashion, then the check for odd makes sense.
If on the other hand you really want the single truthy argument, then I think the check for one makes sense.

I assumed we wanted the second case (single truthy argument), but maybe I assumed wrong.